client/ide/lib/index.coffee

```diff
@@ -549,52 +533,46 @@ class IDEAppController extends AppController
 
   createInitialView: (withFakeViews) ->
```

Comment:
it's becoming shorter actually it was https://github.com/koding/koding/blob/master/client/ide/lib/index.coffee#L550-L597 (47 lines) but now https://github.com/koding/koding/blob/d8c0fd4522a2e8223437930943602aa8f110ab69/client/ide/lib/index.coffee#L534-L575 (41 lines) :) but I can split it out of course.